File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.js
function solution(n) {
    var answer = 0;
    
    for(let i = 1 ; i <= Math.sqrt(n) ; i++)
        if(n % i === 0){
            answer++;
            if(i !== parseInt(n / i))
                answer++;
        }
    
    return answer;
}

File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.java
class Solution {
    
    public int solution(int n) {
        
        int cnt = 0;
        
        for(int i = 1 ; i <= (int)Math.sqrt(n) ; i++)
            if(n % i == 0){
                cnt++;
                if(n / i != i)
                    cnt++;
            }
        
        return cnt;
    }
}